pkg/pillar/agentlog/agentlog.go

```diff
@@ -28,14 +30,38 @@ var savedAgentName string = "unknown" // Keep for signal and exit handlers
 func initImpl(agentName string, logdir string, redirect bool,
 	text bool) (*os.File, error) {
```

Comment:
can you please document the desired interplay between text and redirect flags? Just add a comment right before the initImpl.

Reply:
will do.

Reply:
committed changes. Can you please take a look and let me know?